virttest/qemu_vm.py

```diff
@@ -1694,6 +1709,13 @@ def sort_key(dev):
                                     numa_mem, numa_cpus, numa_nodeid)
             devices.insert(StrDev('numa', cmdline=cmdline))
 
```

Comment:
I would prefer to judge if `devices.has_option("numa dist,.*")` here, not necessary to judgement repeatedly in the for loop.

Reply:
A good catch, let me move it out, thanks!